### PR TITLE
add basic PinyinVariant info

### PR DESCRIPTION
**Pull request by shawn111**
Add some information about PinyinVariant

### CODE REVIEW

File: src/editor/zhuyin_layout/pinyin.rs [hunk 1]
```diff
@@ -12,16 +12,19 @@ const MAX_PINYIN_LEN: usize = 10;
 /// TODO: docs
 #[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
 pub enum PinyinVariant {
-    /// TODO: docs
+    /// [Hanyu Pinyin](https://en.wikipedia.org/wiki/Pinyin), or simply pinyin
     #[default]
     HanyuPinyin,
-    /// TODO: docs
+    /// [THL, 台灣華語羅馬拼音](https://pinyin.thl.tw/)
```
Comment (jserv) [inline src/editor/zhuyin_layout/pinyin.rs:18]: Don't use Chinese characters in comments unless you have to.

Comment (shawn111) [reply src/editor/zhuyin_layout/pinyin.rs]: /// [THL, taíwan huáyǔ luómǎ pinyin](https://pinyin.thl.tw/)

Comment (kanru) [reply src/editor/zhuyin_layout/pinyin.rs]: It looks better with title case `Taíwan Huáyǔ Luómǎ Pinyin`

Comment (shawn111) [reply src/editor/zhuyin_layout/pinyin.rs]: updated

**Review by kanru — APPROVED** [review]

**Pull request merged**